### ISSUE TITLE
NF: Add ability to display extractor-specific docstring via CLI

### ISSUE THREAD

**Issue by jsheunis** [datalad/datalad-metalad#368]
I would like to be able to display the docstring of the main class for an extractor (the class that is specified as the entrypoint) via the command line, ideally with a `--help` flag. For example:

```
datalad meta-extract metalad_studyminimeta --help
```

This would improve UX compared to the current situation where I have to visit the docs whenever I want to know something about a specific extractor.

I assume we'd have to introduce a new argument, similar to `--pipeline-help` in `meta-conduct` (see [here](https://github.com/datalad/datalad-metalad/blob/master/datalad_metalad/conduct.py#L160)), since `--help` is reserved for the docstring of the main interface being called.

@christian-monch could you let me know your thoughts? I can work on a PR later.

**Comment by christian-monch** [comment]
Good idea. It would be great if `--help` was context-aware and would refer to `metalad_studyminimeta` instead of `meta-extract`. I think that might be tricky, but that is just a hunch.

We could use a `--extractor-help` command instead.
